slick/src/main/scala/slick/jdbc/JdbcProfile.scala

```diff
@@ -62,6 +62,8 @@ trait JdbcProfile extends SqlProfile with JdbcActionComponent
       new JdbcActionExtensionMethods[E, R, S](a)
 
     implicit def actionBasedSQLInterpolation(s: StringContext): ActionBasedSQLInterpolation = new ActionBasedSQLInterpolation(s)
+
+    def unsafeSQL(sql: String): SQLActionBuilder = new SQLActionBuilder(Seq(sql + " "), SetParameter.SetUnit)
```

Comment:
To ensure that this does not fail
```
val qHead = sql"SELECT * from table"
val qTail = sql"WHERE id = 5"
val query = (qHead ++ qTail).as[Data]
```

In case users forget to add a space when concatenating queries.